lib/http/options.rb

```diff
@@ -50,7 +50,8 @@ def initialize(options = {})
         :keep_alive_timeout => 5,
         :headers            => {},
         :cookies            => {},
-        :encoding           => nil
+        :encoding           => nil,
+        :logger             => HTTP::Logger.new
```

Comment:
A agree. It should be either `nil` (I like this idea better), or at least some kind of cached null-logger.